addon/components/project-form/date.hbs

```diff
@@ -0,0 +1,10 @@
+<PikadayInput
+  @format="DD.MM.YYYY"
+  @placeholder={{t "ember-gwr.projectForm.datePlaceholder"}}
+  @value={{@value}}
+  @onSelection={{@update}}
+  class="uk-width-1 {{configured-classes "date-picker"}}"
```

Comment:
This `uk-width-1` and all others would have to come from the `configured-classes`.